### PR TITLE
Small table cell tooltip changes

### LINKED ISSUE #1954: Small issues with the table tooltips
- [x] Remove prefixes (Metrics, Param) - they are not needed yet in our case
- [x] Can we try to keep tooltips if I hover on it?
- [ ] Bug with booleans (check strings, arrays?): 

Screen Shot 2022-06-27 at 7 31 54 PM

### PR DESCRIPTION
**Pull request by rogermparent**
Fixes #1954

https://user-images.githubusercontent.com/9111807/176987842-41bc3542-e4ab-469e-b0dc-8229c154c78c.mp4

- [x] Remove prefixes (Metrics, Param) - they are not needed yet in our case.
- [x] Table cell tooltips have `interactive` set on them, so users can do actions like copying text. Currently has a bug where clicking the tooltip in any way registers as a row click for selection.
- [x] Tooltip values are cast to `String`, confirmed booleans, arrays, and 0 work.

note: seems like `null`, even when put into yaml, breaks table parsing.

### CODE REVIEW

File: webview/src/experiments/util/buildDynamicColumns.tsx [hunk 1]
```diff
@@ -58,10 +38,11 @@ const Cell: React.FC<Cell<Experiment, string | number>> = cell => {
   return (
     <Tooltip
       className={tooltipStyles.padded}
-      content={<CellTooltip cell={cell} />}
+      content={String(value)}
```
Comment (rogermparent) [inline webview/src/experiments/util/buildDynamicColumns.tsx:41]: We'll need a component when we display anything more complex, but with this specific implementation for label-less tooltips it's never been less justified to have a separate component.

Comment (rogermparent) [reply webview/src/experiments/util/buildDynamicColumns.tsx]: Turns out "anything more complex" also includes adding `stopPropagation` so selection clicks don't register as row clicks.

File: webview/src/shared/components/tooltip/styles.module.scss [hunk 2]
```diff
@@ -17,7 +17,7 @@ $font: var(--vscode-font-family);
   white-space: nowrap;
   & > :global(.tippy-arrow) {
     color: $background;
-    z-index: 2;
+    z-index: 1;
```
Comment (rogermparent) [inline webview/src/shared/components/tooltip/styles.module.scss:20]: Fixes a small issue with the arrow appearing over text highlighting in content

**Comment by shcheklein** [comment]
@rogermparent is it still a draft?

**Review by wolmir — APPROVED** [review]

**Comment by rogermparent** [comment]
I wanted to write some real tests for the other tooltip data types, but I could do that as an immediate follow-up as well to get this fix in since it's at better tested and working than what we currently have.

**Review by wolmir — APPROVED** [review]
I'm ok with adding the tests as a follow-up

**Comment by mattseddon** [comment]
Please do follow up.

**Pull request merged**